src/PriceCalculatorKata/Program.cs

```diff
@@ -1,12 +1,66 @@
 ﻿using System;
-
+using System.Collections.Generic;
+using System.Globalization;
 namespace PriceCalculatorKata
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var product = new Product()
+            {
+                Name = "The Little Prince",
+                Price = 20.25,
+                UPC = 12345,
+                UPCDiscountPercentage = 7,
+            };
+            var discountCalculations = new DiscountCalculations()
```

Comment:
lets now talk about these calculation classes :)
we have multiple ones that seem to have some common logic. How about to use interfaces and abstract classes ? try to refactor to have some common methods maybe ? give it a try ;)

Reply:
and notice that Product property is also common, coincidence ? i guess you not :3

Reply:
Thank You :)
I have improved the code and you can check for it now ;)